src/main/java/frc/robot/DriveTrain.java

```diff
@@ -1,78 +1,75 @@
 package frc.robot;
 
 /*----------------------------------------------------------------------------*/
+
 /* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
 /* Open Source Software - may be modified and shared by FRC teams. The code   */
 /* must be accompanied by the FIRST BSD license file in the root directory of */
 /* the project.                                                               */
 /*----------------------------------------------------------------------------*/
 
-import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
-import com.ctre.phoenix.sensors.PigeonIMU;
-import com.revrobotics.CANSparkMax;
-
 import edu.wpi.first.wpilibj.drive.DifferentialDrive;
-import edu.wpi.first.wpilibj.DoubleSolenoid;
+import com.revrobotics.CANSparkMax;
 import edu.wpi.first.wpilibj.Joystick;
 import edu.wpi.first.wpilibj.SpeedControllerGroup;
-import edu.wpi.first.wpilibj.XboxController;
 import edu.wpi.first.wpilibj.command.Subsystem;
 
 public class DriveTrain extends Subsystem {
   private HoloTable holo = HoloTable.getInstance();
-  private WPI_TalonSRX driveTurn;
   private CANSparkMax leftFront;
   private CANSparkMax rightFront;
   private CANSparkMax leftRear;
   private CANSparkMax rightRear;
   private DifferentialDrive differentialDrive;
-  private PigeonIMU gyro;
-  private XboxController controller;
-  private DoubleSolenoid turnSolenoid;
+  //private DoubleSolenoid turnSolenoid;
   private SpeedControllerGroup leftMotors;
   private SpeedControllerGroup rightMotors;
   private Joystick joystickLeft;
   private Joystick joystickRight;
+  private double driveCoeff;
 
   /**
    * Creates a new DriveTrain.
    */
   public DriveTrain() {
-    driveTurn = holo.getDriveTurn();
     leftFront = holo.getDriveLeftFront();
     rightFront = holo.getDriveRightFront();
     leftRear = holo.getDriveLeftRear();
     rightRear = holo.getDriveRightRear();
-    gyro = holo.getGyro();
-    controller = holo.getController();
     joystickLeft = holo.getJoystickLeft();
     joystickRight = holo.getJoystickRight();
-    turnSolenoid = holo.getTurnSolenoid();
+    //turnSolenoid = holo.getTurnSolenoid();
     leftMotors = new SpeedControllerGroup(leftFront, leftRear);
     leftMotors.setInverted(true);
     rightMotors = new SpeedControllerGroup(rightFront, rightRear);
     rightMotors.setInverted(true);
     differentialDrive = new DifferentialDrive(leftMotors, rightMotors);
+    driveCoeff = 1;
   }
 
+
   @Override
   public void periodic() {
 
     // This method will be called once per scheduler run
   }
 
   public void driveTank() {
-    differentialDrive.tankDrive(joystickLeft.getY(), joystickRight.getY(), true);
-  }
+    if(joystickRight.getRawButton(2))
+      driveCoeff = .3;
+    else
+      driveCoeff = 1;
+      differentialDrive.tankDrive(driveCoeff * Math.pow(joystickLeft.getY(), 3 ) * 0.5, driveCoeff * Math.pow(joystickRight.getY(), 3 ) * 0.5, false);
```

Comment:
I think this indent on line 62 should not be there. It won't affect execution, but it's a little off-putting because one could mistaken it to be part of the else statement and add curly braces.